### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Select Sunscriptions_Keyboard: Keyboard focus is not staying on the "Select Subscriptions" button when we close the "Subscriptions" pop up.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2677]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#Azure Tools for VS Code; #A11ySev2; #A11yTCS; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Keyboard; #WCAG2.4.3; #Win10;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57c7
Date: 2021-02-08T23:29:42.785Z
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
4. Tab till "Install" button and press enter on it.
5. Tab till "Azure" button from left side pane and press enter on it.
6. Default screen will get open.
7. Tab to Tree view and use down arrow to navigate to tree items.
8. Tab to "Select Subscriptions" and press enter on it.
9. Navigate to list items and press "Escape" to close pop up.
10. Verify focus order.

### Actual Result:
Keyboard focus is not staying on the "Select Subscriptions" button when we close the "Subscriptions" pop up.

### Expected Result:
Keyboard focus should stay on the "Select Subscriptions" button when we close the "Subscriptions" pop up.

### User Impact:
Keyboard users find difficulty in navigating to controls if focus order is not proper.
### Attachment for Reference:

[2677_Select Subscriptions_Keyboard issue.zip](https://github.com/microsoft/vscode-azurefunctions/files/6019980/2677_Select.Subscriptions_Keyboard.issue.zip)

**Comment by ejizba** [comment]
Extensions do not control focus behavior - this should be filed directly on VS Code.
